docs/source/tools/visual.rst

```diff
@@ -0,0 +1,11 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Visualizing DAML Contract
```

Comment:
and you still need the underline otherwise it's not a title!

Reply:
I am assuming ### is used for title correct me if I am wrong, please